roles/openshift_master/tasks/main.yml

```diff
@@ -249,7 +249,7 @@
   # Using curl here since the uri module requires python-httplib2 and
   # wait_for port doesn't provide health information.
   command: >
-    curl --silent
+    curl --silent --tlsv1.2
```

Comment:
All OpenShift versions support TLS v1.2 regardless of the minimum version that's now configurable in 3.5. This is working around a bug in Curl from RHEL 7.1 that did not properly negotiate up to v1.2, so I think it's safe.